paymentsheet/src/main/java/com/stripe/android/paymentsheet/wallet/SavedPaymentMethodsSheet.kt

```diff
@@ -0,0 +1,74 @@
+package com.stripe.android.paymentsheet.wallet
+
+import androidx.activity.ComponentActivity
+import androidx.annotation.RestrictTo
+import androidx.fragment.app.Fragment
+import com.stripe.android.ExperimentalSavedPaymentMethodsApi
+import com.stripe.android.paymentsheet.PaymentSheet
+import com.stripe.android.paymentsheet.repositories.CustomerAdapter
+
+/**
+ * 🏗 This feature is under construction 🏗
+ *
+ * [SavedPaymentMethodsSheet] A class that presents a bottom sheet to manage a customer's
+ * saved payment methods.
+ */
+@ExperimentalSavedPaymentMethodsApi
+@RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
+class SavedPaymentMethodsSheet {
+
+    @Suppress("UNUSED_PARAMETER", "unused")
+    constructor (
```

Comment:
I still like the idea of making these factory methods. It allows us to create subclasses, change this to an interface, and have nicer/different names in the future if we want them.

It also allows us to create this thing with dependency injection and have an internal constructor.